public/beach_style.css

```
@@ -46,8 +46,9 @@ body {
   position: absolute;
 }
 
-.InsectNet {
-  z-index: 1;
+.InsectNet{
+    z-index: 1;
+    transition-duration: 500ms;
```

```suggestion
.InsectNet{
  z-index: 1;
  transition-duration: 500ms;
```

他の記述とインデントをあわせてほしいです。